lasagne/layers/conv.py

```diff
@@ -1127,8 +1126,16 @@ def convolve(self, input, **kwargs):
         conved = op(self.W, input, output_size)
         return conved
 
+
 Deconv3DLayer = TransposedConv3DLayer
 
+if not hasattr(T.nnet.abstract_conv, 'AbstractConv3d_gradInputs'):
+    # pragma: no cover
```

Comment:
The `# pragma: no cover` must be on the same line as the `if` case. It's caught by a regular expression that ignores all lines which have this string in them. The following should work:
```python
if not hasattr(T.nnet.abstract_conv,
               'AbstractConv3d_gradInputs'):  # pragma: no cover
```